x11-misc/libinput-gestures/libinput-gestures-2.34.ebuild

```diff
@@ -5,15 +5,20 @@ EAPI=6
 
 PYTHON_COMPAT=( python{3_4,3_5,3_6} )
 
-inherit python-single-r1
+inherit gnome2-utils python-single-r1
 
 DESCRIPTION="Actions gestures on your touchpad using libinput"
-HOMEPAGE="https://github.com/bulletmark/${PN}"
-SRC_URI="https://github.com/bulletmark/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://github.com/bulletmark/libinput-gestures"
+if [[ "${PV}" = 9999 ]] ; then
```

Comment:
`if [[ ${PV} == *9999 ]] ; then`

Reply:
Why `*9999`?

Reply:
because it generalises to say `2.1.9999` branch "live" versions

Reply:
FIxed...